Makefile

```diff
@@ -0,0 +1,33 @@
+DOCKER_IMAGE = oam/server:latest
+
+all: server
+
+server:
+	@docker build -f ./server/Dockerfile -t $(DOCKER_IMAGE) ./server
```

Comment:
What version of Docker are you using? I'm on 1.4.1, and it's complaining that the `-f` flag doesn't exist.